ci/ci/build.py

```diff
@@ -842,34 +848,50 @@ def config(self, scope):  # pylint: disable=unused-argument
         return {
             'token': self.token,
             'name': self._name,
-            'admin_username': self.admin_username,
             'admin_secret_name': self.admin_secret_name,
-            'user_username': self.user_username,
             'user_secret_name': self.user_secret_name
         }
 
+    def build_cant_create_database(self, batch, code, scope):  # pylint: disable=unused-argument
+        script = f'''
+kubectl -n {shq(self.namespace)} create secret generic {shq(self.admin_secret_name)} --from-file=/sql-config/sql-config.json --from-file=/sql-config/sql-config.cnf
```

Comment:
I just saw your other PR where there are delete secrets first. Do you need those here?

Reply:
I think it could fail in dev deploy without, yeah, done.